### ISSUE TITLE
<QueueManager> No manifest.json file found 'custom_components/None/manifest.json' after updating to 0.5.0

### ISSUE THREAD

**Issue by SanderKo85** [jeatheak/Mitsubishi-WF-RAC-Integration#73]
Hi!

I don't know exactly what happend, but got Lovelace and HACS errors after updating to 0.5.0.

`<QueueManager> No manifest.json file found 'custom_components/None/manifest.json'`

![repository errors wf-rac](https://github.com/jeatheak/Mitsubishi-WF-RAC-Integration/assets/119807149/bb5d93c9-007b-4d3f-abe3-c57208b77964)

```
Logger: homeassistant.components.websocket_api.http.connection
Source: custom_components/hacs/repositories/integration.py:173
Integration: Home Assistant WebSocket API ([documentation](https://www.home-assistant.io/integrations/websocket_api), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+websocket_api%22))
First occurred: 14:46:57 (1 occurrences)
Last logged: 14:46:57
[140646596926400] Error handling message: Unknown error (unknown_error) Sander Koorneef from 89.31.56.76 (Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:121.0) Gecko/20100101 Firefox/121.0)

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/decorators.py", line 26, in _handle_async_response
    await func(hass, connection, msg)
  File "/config/custom_components/hacs/websocket/repository.py", line 232, in hacs_repository_download
    await repository.update_repository(force=True)
  File "/config/custom_components/hacs/utils/decorator.py", line 28, in wrapper
    result = await function(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hacs/repositories/integration.py", line 126, in update_repository
    if manifest := await self.async_get_integration_manifest():
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hacs/repositories/integration.py", line 173, in async_get_integration_manifest
    raise HacsException(f"No {RepositoryFile.MAINIFEST_JSON} file found '{manifest_path}'")
custom_components.hacs.exceptions.HacsException: No manifest.json file found 'custom_components/None/manifest.json'
```

and

```
Logger: custom_components.hacs
Source: custom_components/hacs/websocket/repository.py:48
Integration: HACS ([documentation](https://hacs.xyz/docs/configuration/start), [issues](https://github.com/hacs/integration/issues))
First occurred: 14:46:54 (1 occurrences)
Last logged: 14:46:54
<Integration jeatheak/Mitsubishi-WF-RAC-Integration> No manifest.json file found 'custom_components/None/manifest.json'
```

I am trying to figure out why now.

**Comment by jeatheak** [comment]
from which version did you update?

**Comment by SanderKo85** [comment]
Hi! From 0.4.0.
Will check the location, but always used the hacs version

**Comment by SanderKo85** [comment]
Fixed after OS update from 11.3 to 11.4, real reason unknown.

**Comment by jeatheak** [comment]
Ow that's weird indeed. Glad it did fix it. I will try to check if I can find something.
